src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Log4NetLogging/Log4netWrapper.cs

```diff
@@ -18,7 +18,8 @@ public class Log4netWrapper : IWrapper
         private static Func<object, string> _getRenderedMessage;
         private static Func<object, DateTime> _getTimestamp;
         private static Func<object, Exception> _getLogException;
-        private static Func<object, IDictionary> _getProperties;
+        private static Func<object, IDictionary> _getGetProperties; // calls GetProperties method
+        private static Func<object, IDictionary> _getProperties; // getter for Properties property
```

Comment:
It doesn't look like this is being used?

Reply:
We are using both _getGetProperties and _getProperties.
_getGetProperties  is in GetContextData
_getProperties is in DecorateLogMessage

Reply:
Ah! My mistake.